CHECKLIST.md

```
@@ -1,5 +1,35 @@
 # Coeus Development Roadmap Checklist
 
+## ATLAS-COEUS-SAFETY-003 Uninitialized COW replacement [arch][minor][perf]
```

markdownlint-cli2 (0.23.0)

[warning] Reference links and images should use a label that is defined
Missing link or image reference definition: "perf"

(MD052, reference-links-images)